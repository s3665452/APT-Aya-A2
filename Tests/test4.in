YULRLRBRULYUULURBLUYUBRLUYBLYRBLUYRLBYBULYBRYLBUYLBYRULBYYUBLYRBULYBYUBLRYRLYUBRLYBRLURYLRRRBBURBURU
Mika
Isis
turn 5 L 1
turn 4 U 2
turn 3 U 2
turn 0 U 1
turn 2 R 3
turn 0 L 3
turn 1 Y 4
turn 0 B 4
turn 0 Y 4
turn 0 R 5
turn 0 U 5