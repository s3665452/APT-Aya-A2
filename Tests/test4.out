Factories:
0: L  
1: 
2: 
3: 
4: 
5: 

Score for Mika: 0
Mosaic for Mika:
1:         L || . . . . .
2:       U U || . . . . .
3:     . R R || . . . . .
4:   . Y Y Y || . . . . .
5: . . . . U || . . . . .
broken:

Score for Player2: 0
Mosaic for Player2:
1:         U || . . . . .
2:       U U || . . . . .
3:     L L L || . . . . .
4:   . . B B || . . . . .
5: . . . R R || . . . . .
broken: F

turn 5 L 1
turn 4 U 2
turn 3 U 2
turn 0 U 1
turn 2 R 3
turn 0 L 3
turn 1 Y 4
turn 0 B 4
turn 0 Y 4
turn 0 R 5
turn 0 U 5
